### PR TITLE
Added Sea Region

### LINKED ISSUE #715: SEA routing for OCE
If this change will not be made, fetching oce games would be not possible

### PR DESCRIPTION
**Pull request by AntonioZn**
Fixes https://github.com/BenFradet/RiotSharp/issues/715 and https://github.com/BenFradet/RiotSharp/issues/719

### CODE REVIEW

File: RiotSharp/Misc/Region.cs [hunk 1]
```diff
@@ -102,6 +102,11 @@ public enum Region
         /// <summary>
         /// Latin America.
         /// </summary>
-        Latam
+        Latam,
+
+        /// <summary>
+        /// Sount East Asia.
```
Comment (JanOuborny) [inline RiotSharp/Misc/Region.cs]: Small typo, should be `Southeast Asia`

Comment (AntonioZn) [reply RiotSharp/Misc/Region.cs]: fixed

Comment (AntonioZn) [reply RiotSharp/Misc/Region.cs]: @JanOuborny  can you approve it please?

Comment (AntonioZn) [reply RiotSharp/Misc/Region.cs]: @BenFradet can you take a look please?

**Comment by fbesteiro-duetto** [comment]
It would be extremely useful to get this merged. @JanOuborny if you have a chance, could you approve it?

**Comment by Renkon** [comment]
Please, could it be possible to get this approved? @JanOuborny it would be extremely useful if you have some time available.

**Review by BenFradet — APPROVED** [review]
Thanks :+1:

**Pull request merged**